frameworks/svelte/package.json

```diff
@@ -38,8 +38,9 @@
   "devDependencies": {
     "@babel/preset-env": "^7.12.13",
     "@babel/preset-typescript": "^7.12.13",
-    "@cloudinary/base": "^1.0.0-beta.2",
+    "@cloudinary/base": "^1.0.0-beta.3",
```

Comment:
@newmaniese Can you please remove the version bumps? we have an automation that takes care of this:)